### ISSUE TITLE
master thieve's armband

### ISSUE THREAD

**Issue by Xazmadan** [egoal/darkest-pixel-dungeon#17]
armband of thieve doesnt count midas and evil greed transformation gold. Idk is it a bug or it is planned, so i created issue

**Comment by egoal** [comment]
no, I checked the code, the normal version of the claw should count. 

the evil, however, wont trigger the armband.

now I modified this, both version should trigger the armband in the next release.
